environment.yml

```diff
@@ -18,6 +18,7 @@ dependencies:
   # - sleap::pyside2=5.14.1
   - qtpy>=2.0.1
   - conda-forge::pip!=22.0.4
+  - pip
```

Comment:
Did you experience any issues with this removed?

Reply:
Old yaml gave infinite warnings suggesting some packages require `pip`, without `pip` installation cannot be done. Adding pip solved the problem, so was environment_build.yml.  environment_m1.yml has pip line though.